templates/facade.js

```diff
@@ -88,4 +85,37 @@ if (wrappers.wrap{{ name }}) {
   {{ name }}V{{ version }} = wrappers.wrap{{ name }}({{ name }}V{{ version }});
 }
 
+/**
```

Comment:
I think this should live in its own js file, whis is then imported and used here. There not seem to be anything requiring a template for this function, it's justa. generic utility, maybe it should live in /api/transform.js